dist/dfu.js

```diff
@@ -134,6 +134,9 @@
             })
             .then(function() {
                 log("modeData written");
+                setTimeout(function() {  // TODO: Remove this when gattserverdisconnected event is implemented and possibly put a timeout in that event handler before resolving.
```

Comment:
The node example uses the bleat library which implements the web bluetooth API on top of the node noble library.
Problem here is that the bleat implementation has the disconnect event implemented while the chrome implementation hasn't.

This is how I've had the DFU working end to end for a while now, recommend you try it :)